### DISCUSSION TITLE
How did you know how to read the .enpassdbsync file?

### DISCUSSION THREAD

**Discussion by mabasic** [hazcod/enpass-cli#119] [General]
Is there somewhere a specification on how enpass treats this file?

I've just skimmed the code, but it seems like it is a encrypted sqlite database?

I find this application very useful in case enpass goes down. Thank you for building this.

**Comment by hazcod** [comment]
Hi @mabasic , I based myself upon the [enpass security whitepaper ](https://support.enpass.io/docs/security-whitepaper-enpass/index.html)and the sqlcipher spec.

**Reply by mabasic** [reply]
Thank you for the answer ❤️
